src/xbot/common/command/scripted/functions/RequireCommandsFunction.java

```diff
@@ -0,0 +1,30 @@
+package xbot.common.command.scripted.functions;
+import java.util.Arrays;
+import java.util.function.Consumer;
+
+import org.mozilla.javascript.Context;
+import org.mozilla.javascript.Scriptable;
+
+/**
+ * A JavaScript-callable function to allow scripts to specify that they require access
```

Comment:
Could elaborate a bit more on what this require access thing is about? Conceptually any command you plan to use in the script?